### PR TITLE
fix calculation of lease expiration for small numbers (should start from...

### PR DESCRIPTION
**Pull request by domsj**
... a float)

### CODE REVIEW

File: src/paxos/multi_paxos.ml [hunk 1]
```diff
@@ -229,16 +229,21 @@ let start_lease_expiration_thread (type s) constants =
...
-      then
-        (* sleep a little longer than necessary on a slave
-           to prevent a node thinking it's still the master
-           while some slaves have elected a new master amongst them
-           (in case the clocks don't all run at the same speed) *)
-        (float_of_int constants.lease_expiration) *. 1.1
-      else
-        float_of_int (constants.lease_expiration / 2) in
+      lease_expiration *.
+        begin
```
Comment (toolslive) [inline src/paxos/multi_paxos.ml]: (\* comment ... *)
let factor = if slave then ... else ... in
let sleep_sec = lease_expiration *. factor
might be easier on the brain.

Comment (domsj) [inline src/paxos/multi_paxos.ml]: yes, good point. I'll change it.

**Comment by domsj** [comment]
updated

**Comment by toolslive** [comment]
fine.

**Pull request merged**